pkg/pb/metainfo.proto

```diff
@@ -450,8 +450,9 @@ message SegmentBeginDeleteRequest {
 }
 
 message SegmentBeginDeleteResponse {
-    bytes segment_id = 1 [(gogoproto.customtype) = "SegmentID", (gogoproto.nullable) = false];
-    repeated AddressedOrderLimit addressed_limits = 2;
+    bytes                           segment_id = 1 [(gogoproto.customtype) = "SegmentID", (gogoproto.nullable) = false];
+    repeated AddressedOrderLimit    addressed_limits = 2;
+    bytes                           private_key = 3 [(gogoproto.customtype) = "PiecePrivateKey", (gogoproto.nullable) = false];
```

Comment:
Why do we need the private key for deleting?

Reply:
hmm you tell me:) piecestore client has it as a parameter for deleting. It was added with new order signing thing.

Reply:
Ah, that's the piece private key... I was thinking it was something else. Yeah, this is mainly for consistency with all other things.